string garbage
---
nav_msgs/OccupancyGrid map